### PR TITLE
Pull request generation from completed recipe

### PR DESCRIPTION
**Pull request by JLLeitschuh**
Signed-off-by: Jonathan Leitschuh <Jonathan.Leitschuh@gmail.com>

### CODE REVIEW

File: src/omega_moderne_client/campaign/campaign_executor.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ async def launch_pull_request(
 
     async def await_pull_request(self, commit_id: str):
         while True:
-            job_state = await self.client.query_commit_job_with_summary(commit_id)
+            job_state = await self.client.query_commit_job_status(commit_id)
```
Comment (Cyber-JiuJiteria) [inline src/omega_moderne_client/campaign/campaign_executor.py:68]: Why the change of function? What did you gain from status v summary?

**Review by Cyber-JiuJiteria — APPROVED** [review]

File: src/omega_moderne_client/campaign/campaign_executor.py (hunk 1 above)
Comment (JLLeitschuh) [reply src/omega_moderne_client/campaign/campaign_executor.py:68]: New method is a different GraphQL call that requests more information. I should probably delete the other/old method soon
